vdmvdo.go

```diff
@@ -0,0 +1,77 @@
+package nmea
+
+const (
+	// TypeVDM type for VDM sentences
+	TypeVDM = "VDM"
+
+	// TypeVDO type for VDO sentences
+	TypeVDO = "VDO"
+)
+
+// VDMVDO is a format used to encapsulate generic binary payloads. It is most commonly used
+// with AIS data.
+// http://catb.org/gpsd/AIVDM.html
+type VDMVDO struct {
+	BaseSentence
+	NumFragments   int64
+	FragmentNumber int64
+	MessageID      int64
+	Channel        string
+	Payload        []byte
+	FillBits       int64
+}
+
+// decodePayload will remove the 6-bit ascii armor
+func decodePayload(payload []byte, fillBits int) []byte {
```

Comment:
Make this a method on the parser type.

``` go
func (p *parser) AsciiArmour(i int, fillBits int) []byte
```

Reply:
Should we name the method VDMVDOAsciiArmor?

Some devices use a different ascii armor in other messages.

Reply:
Yeah, unless there's a more generic name. `SixBitAsciiArmour`? Or are there different variations of that?